insights/parsers/docker_sysconfig.py

```diff
@@ -50,7 +51,9 @@ class DockerSysconfig(SysconfigOptions):
 
     Parse the ``/etc/sysconfig/docker`` file.
     """
-    pass
+    def __init__(cls, *args, **kwargs):
```

Comment:
I would expect to see `self` instead of `cls` as the first argument here.

Reply:
There is a deprecated function in this module, would it make sense to also add the deprecation functionality to it as well since you are modifying this module?